controller/relay-image.go

```diff
@@ -24,26 +26,27 @@ func relayImageHelper(c *gin.Context, relayMode int) *OpenAIErrorWithStatusCode
 	group := c.GetString("group")
 
 	var imageRequest ImageRequest
+
 	if consumeQuota {
 		err := common.UnmarshalBodyReusable(c, &imageRequest)
 		if err != nil {
 			return errorWrapper(err, "bind_request_body_failed", http.StatusBadRequest)
 		}
 	}
 
-	// Prompt validation
-	if imageRequest.Prompt == "" {
-		return errorWrapper(errors.New("prompt is required"), "required_field_missing", http.StatusBadRequest)
+	// Model validation
+	if imageRequest.Model != "" && imageRequest.Model != "dall-e-3" {
```

Comment:
The lack of assigning the model name parsed from `UnmarshalBodyReusable` to `imageModel` here causes it to always be `dall-e2` later on.

Reply:
And I think return a `invalid_field_value` error when `imageRequest.Model` is neither `dall-e-2` nor `dall-e-3`, will be better than assign default `dall-e-2` here